### DISCUSSION TITLE
Import Issues

### DISCUSSION THREAD

**Discussion by johnkaryu** [niftools/blender_niftools_addon#654] [Q&A]
Hi guys! I work as 3D artist and we use 3DS Max with some Nif exportation tools.
Since found this tool for Blender, I would like to try get back to Blender, but I'm facing some issues, if someone can help me, I would be very gratetul.
Like this: I'm trying to import a Nif file from Digimon Masters game, but I got a lot of this errors, and it doesn't bring materials information or stuff like that. 
I already read the docs, am I missing something?
Thanks a lot!
![image](https://github.com/user-attachments/assets/8a0b5571-2e26-4948-a36c-8f72d348574e)

**Comment by TheCraftyLemur** [comment]
I am having a similar import issue, although the error is slightly different 
![image](https://github.com/user-attachments/assets/7eed9fb0-fcdd-4f20-9280-eae55e92a846)

**Reply by wakuwakumafia** [reply]
I'm also having this issue, even with a clean install.
